commit 87445a20525bfaa082bbde94cd4b78ffec378157
Author: Bertram Felgenhauer <int-e@gmx.de>
Date:   Sat Aug 22 17:44:06 2015 +0200

    utf8-encode output from compiler (#131)
    
    This reinstates commit 48861d7504c6f06a7b5b5efa2d8654c1cc5a073d.

diff --git a/lambdabot-haskell-plugins/src/Lambdabot/Plugin/Haskell/Eval.hs b/lambdabot-haskell-plugins/src/Lambdabot/Plugin/Haskell/Eval.hs
index fc8caac..ab08683 100644
--- a/lambdabot-haskell-plugins/src/Lambdabot/Plugin/Haskell/Eval.hs
+++ b/lambdabot-haskell-plugins/src/Lambdabot/Plugin/Haskell/Eval.hs
@@ -83,8 +83,8 @@ runGHC src = do
     case (out,err) of
         ([],[]) -> return "Terminated\n"
         _       -> do
-            let o = munge out
-                e = munge err
+            let o = mungeEnc out
+                e = mungeEnc err
             return $ case () of {_
                 | null o && null e -> "Terminated\n"
                 | null o           -> e
